test/lib/rubocop/primer_octicon_test.rb

```diff
@@ -273,4 +273,12 @@ def test_medium_size
 
     assert_correction "primer_octicon(:icon, size: :medium)"
   end
+
+  def test_no_args
+    investigate(cop, <<-RUBY)
+      octicon
+    RUBY
+
+    assert_correction "primer_octicon"
+  end
```

Comment:
It looks like the cop explicitly does nothing if no arguments are passed to the `octicon` method, so that's why this test is failing 😄